### PR TITLE
Rebranding (FOG>>PXEL|Host>>Node) & Access Control Feature Enhancement

### PR DESCRIPTION
**Pull request by weitheng10**
@peirkern-intel review

### CODE REVIEW

File: bin/installfog.sh [hunk 1]
```diff
@@ -630,6 +630,7 @@ while [[ -z $blGo ]]; do
                     writeUpdateFile
                     linkOptFogDir
                     updateStorageNodeCredentials
+					enableAccessControlPlugin
```
Comment (peirkern-intel) [inline bin/installfog.sh]: formatting.

File: packages/web/lib/fog/page.class.php [hunk 2]
```diff
@@ -457,7 +463,7 @@ public function render()
     public static function getSearchForm()
     {
         global $node;
-        echo '<div class="col-md-3">';
+        echo '<div class="col-md-2">';
```
Comment (peirkern-intel) [inline packages/web/lib/fog/page.class.php:466]: this is the space after search bar? navbar?

File: packages/web/lib/pages/aboutuspage.class.php [hunk 3]
```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Presents the About Us page.
+ *
+ * PHP version 5
+ *
+ * @category AboutUsPage
+ * @package  FOGProject
+ * @author   CES Team <ive.vla.sre.ces@intel.com>
```
Comment (peirkern-intel) [inline packages/web/lib/pages/aboutuspage.class.php]: author -> WT.

File: packages/web/lib/pages/aboutuspage.class.php [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+ */
+/**
+ * Presents the About Us page.
+ *
+ * @category AboutUsPage
+ * @package  FOGProject
+ * @author   CES Team <ive.vla.sre.ces@intel.com>
```
Comment (peirkern-intel) [inline packages/web/lib/pages/aboutuspage.class.php]: why we hv two redundant sections?

File: packages/web/lib/reports/hosts_and_users.report.php [hunk 5]
```diff
@@ -21,7 +21,7 @@ class Hosts_And_Users extends ReportManagementPage
 
     public function file()
     {
-        $this->title = _('FOG Host and Users - Search');
+        $this->title = _('FOG Host and Users') .'- ' . _('Search');
```
Comment (peirkern-intel) [inline packages/web/lib/reports/hosts_and_users.report.php:24]: the effect of these changes?

File: packages/web/management/activatepluginrun.php [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+ *
+ * PHP version 5
+ *
+ * @category Run
+ * @package  FOGProject
+ * @author   Tom Elliott <tommygunsster@gmail.com>
+ * @license  http://opensource.org/licenses/gpl-3.0 GPLv3
+ * @link     https://fogproject.org
+ */
+/**
```
Comment (peirkern-intel) [inline packages/web/management/activatepluginrun.php]: cleanup

File: packages/web/management/other/index.php [hunk 7]
```diff
@@ -123,7 +135,10 @@
     echo '</div>';
     echo '</div>';
 } else {
-    echo '<nav class="navbar navbar-inverse navbar-fixed-top">';
+    //CES_CUSTOMIZATION 20220323 START     
+    //echo '<nav class="navbar navbar-inverse navbar-fixed-top">';
+    //echo '<nav class="navbar navbar-fixed-top" style="background-color: #f8f8f8;border-color: #337AB7">';
```
Comment (peirkern-intel) [inline packages/web/management/other/index.php]: remove

File: bin/installfog.sh (hunk 1 above)
Comment (weitheng10) [reply bin/installfog.sh]: Fix on next release

File: packages/web/lib/pages/aboutuspage.class.php (hunk 3 above)
Comment (weitheng10) [reply packages/web/lib/pages/aboutuspage.class.php]: fixed on next release

File: packages/web/management/activatepluginrun.php (hunk 6 above)
Comment (weitheng10) [reply packages/web/management/activatepluginrun.php]: fixed on next release

File: packages/web/management/other/index.php (hunk 7 above)
Comment (weitheng10) [reply packages/web/management/other/index.php]: removed 140 and remain the ori code 139 on next release

**Comment by weitheng10** [comment]
Fixed on next release will create another pull request

**Review by peirkern-intel — APPROVED** [review]

**Comment by weitheng10** [comment]
@peirkern-intel have complete the fixes kindly review again

**Pull request merged**